app/src/main/java/umc/everyones/lck/util/LoginManager.kt

```diff
@@ -0,0 +1,26 @@
+package umc.everyones.lck.util
+
+import android.content.Context
+import android.content.SharedPreferences
+
+class LoginManager(context: Context) {
+
+    private val sharedPreferences: SharedPreferences = context.getSharedPreferences(PREFS_NAME, Context.MODE_PRIVATE)
+
+    companion object {
+        private const val PREFS_NAME = "umc.everyones.lck.prefs"
+        private const val KEY_IS_LOGGED_IN = "isLoggedIn"
+    }
+
+    fun setLoggedIn(isLoggedIn: Boolean) {
+        sharedPreferences.edit().putBoolean(KEY_IS_LOGGED_IN, isLoggedIn).apply()
+    }
+
+    fun isLoggedIn(): Boolean {
```

Comment:
return 타입이 있으므로 함수명을 이렇게 바꿔보는게 어떨까요??
```suggestion
    fun getIsLoggedIn(): Boolean {
```